lib/matplotlib/backends/backend_pdf.py

```diff
@@ -2072,6 +2078,7 @@ def draw_mathtext(self, gc, x, y, s, prop, angle):
                                      0.001 * fontsize, 0,
                                      0, 0.001 * fontsize,
                                      ox, oy, Op.concat_matrix)
+                    symbol_name = font.get_glyph_name(font.get_char_index(num))
```

Comment:
No, because _stix_fallback._get_glyph(...) will return a different font object (for which the glyph exists).  This can be checked e.g. with
```python
rcParams["mathtext.fontset"] = "cm"; figtext(.5, .5, "$\\perp$"); savefig("/tmp/out.pdf")
```
adding a print to check that fallback occurs.  (Actually it looks like this should *not* fallback as there's a \perp sign in cmsy10.ttf?)